src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -29,5 +38,21 @@ const Footer = BaseFooter.extend({
 
 export default BaseView.extend({
   Body,
-  Footer
+  Footer,
+  createModelClass (currentViewState, optionUiSchemaConfig, settings) {
+    let modelClass = BaseView.prototype.createModelClass.call(this, currentViewState, optionUiSchemaConfig);
```

Comment:
`BaseView.prototype.createModelClass.apply(this, arguments);` ?

Reply:
Didn't want to pass the settings object, but sure, behavior will be the same